packages/react/src/components/Forms/Select/mono-common.tsx

```diff
@@ -174,27 +174,35 @@ export const SelectMonoAux = ({
           {...downshiftReturn.getMenuProps()}
         >
           <ul>
-            {downshiftReturn.isOpen &&
-              options.map((item, index) => {
-                const isActive = index === downshiftReturn.highlightedIndex;
-                return (
-                  <li
-                    className={classNames("c__select__menu__item", {
-                      "c__select__menu__item--highlight": isActive,
-                      "c__select__menu__item--selected":
-                        downshiftReturn.selectedItem === item,
-                      "c__select__menu__item--disabled": item.disabled,
-                    })}
-                    key={`${item.value}${index}`}
-                    {...downshiftReturn.getItemProps({
-                      item,
-                      index,
-                    })}
-                  >
-                    <span>{item.label}</span>
+            {downshiftReturn.isOpen && (
+              <>
+                {options.map((item, index) => {
+                  const isActive = index === downshiftReturn.highlightedIndex;
+                  return (
+                    <li
+                      className={classNames("c__select__menu__item", {
+                        "c__select__menu__item--highlight": isActive,
+                        "c__select__menu__item--selected":
+                          downshiftReturn.selectedItem === item,
+                        "c__select__menu__item--disabled": item.disabled,
+                      })}
+                      key={`${item.value}${index.toString()}`}
```

Comment:
Do we need the index here ? 
Two select option cannot have the same value, or it would be weird.
Here if the liste change, like a element is removed after being selected, react will think that this option change because the key's index is different.

Reply:
let's merge discussion: https://github.com/openfun/cunningham/pull/160#discussion_r1324247385